### ISSUE TITLE
Navigation arrows problem

### ISSUE THREAD

**Issue by acwolff** [fancyapps/fancybox#2228]
If I look with my iPad to your demo page, I see a problem with the previous / next navigation problem, see [here](https://fancyapps.com/fancybox/3/#images-1): fancyBox shows a previous button, but clicking that button has no effect, because there is no previous image.
I think in this case the previous button should be hidden, so the viewer sees he is at the start of the album.
It is the same for the next button of the last image.

**Comment by fancyapps** [comment]
Well, if you do not like current design, you can always change that using CSS (or even hide completely).

**Comment by acwolff** [comment]
Yes I do that already in my [FancyBox skin](https://jalbum.net/en/skins/skin/FancyBox) as you can see [here](https://andrewolff.jalbum.net/FancyBoxWeb/#S-1), but I think this is more a task of the fancyBox nucleus!

**Comment by fancyapps** [comment]
FYI, in future releases, navigation arrows will be hidden by default on mobile devices. I think that people are used that on mobile apps (Twitter, FB) there are no arrows and they know that swiping would navigate to next/prev image.

**Comment by acwolff** [comment]
@fancyapps that is a good idea, but I think this should be an option so that it is still possible to show the arrow buttons on mobile devices.
